app/models/planning_application.rb

```diff
@@ -47,6 +47,9 @@ class WithdrawOrCancelError < RuntimeError; end
     has_many :requests, class_name: "ValidationRequest"
     has_many :assessment_details, -> { by_created_at_desc }, inverse_of: :planning_application
     has_many :permitted_development_rights, -> { order :created_at }, inverse_of: :planning_application
+    has_many :planning_application_constraints
+    # To change to "constraints" when we update the code to use the new association and are able to drop the "constraints" array field
+    has_many :new_constraints, through: :planning_application_constraints, source: :constraint
```

Comment:
We also need to add
```
has_many :planning_application_constraint_queries
```

Reply:
> TODO: After dropping text field array constraints, we should rename the association new_constraints to constraints


We are going to rename the current `constraints` field to `old_constraints` so we don't need to do any further renaming after. This will need to be in 2 prs. 1pr will add an `alias_attribute` and 2nd pr will be the one of doing the renaming.

Reply:
1st pr: https://github.com/unboxed/bops/pull/1097

Reply:
2nd PR: Rename constraints to old_constraints #1099